### PR TITLE
chore: [#160] Add A/B testing entities

### PR DESCRIPTION
**Pull request by christobill**
See linked issue

### CODE REVIEW

File: data/entities.js [hunk 1]
```diff
@@ -330,6 +330,21 @@ module.exports = [
...
+    domains: ['*.useinsider.com'],
+    examples: ['hit.api.useinsider.com'],
+  },
+  {
+    name: 'Adobe Experience Cloud',
+    company: 'Adobe',
+    homepage: '',
+    categories: ['analytics'],
+    domains: ['*.2o7.net', 'du8783wkf05yr.cloudfront.net', '*.hitbox.com', '*.imageg.net', '*.nedstat.com', '*.omtrdc.net'],
+    examples: ['audiag.112.2o7.net'],
```
Comment (patrickhulce) [inline data/entities.js:346]:
```suggestion
    examples: ['audiag.112.2o7.net', 'du8783wkf05yr.cloudfront.net/NS_mbox.js'],
```

File: data/entities.js [hunk 2]
```diff
@@ -6992,13 +6993,6 @@ module.exports = [
     categories: ['ad'],
     domains: ['*.liftdna.com'],
   },
-  {
-    name: 'Opentag',
-    company: 'Qubit',
-    categories: ['tag-manager'],
-    domains: ['*.qutics.com', 'd3c3cq33003psk.cloudfront.net'],
```
Comment (patrickhulce) [inline data/entities.js:6999]: qutics.com seems to have disappeared, add it back to Qubit?

**Review by patrickhulce — COMMENTED** [review]
nice work @christobill thank you!

File: data/entities.js (hunk 2 above)
Comment (christobill) [reply data/entities.js:6999]: Done :heavy_check_mark:

File: data/entities.js (hunk 1 above)
Comment (christobill) [reply data/entities.js]: Done :heavy_check_mark:

**Review by patrickhulce — APPROVED** [review]
thanks so much!

**Pull request merged**